src/libraries/System.Runtime.InteropServices.JavaScript/src/System/Runtime/InteropServices/JavaScript/JSObject.References.cs

```diff
@@ -71,11 +73,17 @@ private void Dispose(bool disposing)
             }
         }
 
+        /// <summary>
+        /// Dispose the JavaScript object handle and release it's reference.
+        /// </summary>
```

Comment:
I don't think it's necessary for a finalizer to have a docstring, it implies you're meant to call it manually when it is there as a last resort